UI/src/view/navigationpanel/ActionPanelBoxes.js

```diff
@@ -169,6 +266,109 @@
     }
   };
 
+  ActionPanelBoxes.ServiceRoadBox = function(asset) {
+    var serviceRoadValues = [
+      [ 0, 'Tieoikeus'],
+      [ 1, 'Tiekunnan osakkuus'],
+      [ 2, 'LiVin hallinnoimalla maa-alueella'],
+      [ 3, 'Kevyen liikenteen väylä'],
+      [ 4, 'Tuntematon']
+    ];
+    var serviceRoadLegendTemplate = _.map(serviceRoadValues, function(serviceRoadValue) {
+      return '<div class="legend-entry">' +
+        '<div class="label">' + serviceRoadValue[1] + '</div>' +
+        '<div class="symbol linear service-road-' + serviceRoadValue[0] + '" />' +
+        '</div>';
+    }).join('');
+
```

Comment:
This uses the `service-road-0` instead of `service-road-Tieoikeus` because it's easier to read the css style. 

Because instead of having for example: 

`  .service-road-2 { .linear-asset-legend-entry(@service-road-2, @linear-asset-node-5); }`~

You would have 

`  .service-road-LiVin hallinnoimalla maa-alueella { .linear-asset-legend-entry(@service-road-LiVin hallinnoimalla maa-alueella, @linear-asset-node-5); }`~